### ISSUE TITLE
scylla_setup fails on 3.2.2  due to embedded python3.8 doesn't  have 'dist' attribute in module [platform]

### ISSUE THREAD

**Issue by pidiaquez** [scylladb/scylla#7755]
This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [x] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

*Installation details*
Scylla version (or git commit hash): 3.2.2-0.20200222.0b23e7145d0
Cluster size: not relevant
OS (RHEL/CentOS/Ubuntu/AWS AMI): Centos 7.9 2009

scylla_setup  script fails on  scylla 3.2.2-0.20200222.0b23e7145d0

command line:

`sudo scylla_setup --nic eth1 --no-raid-setup --no-ntp-setup --no-sysconfig-setup --no-ec2-check`

error:

```
WARN  2020-12-02 21:40:32,905 [shard 0] iotune - Available space on filesystem at /var/tmp/mnt: 124 MB: is less than recommended: 10 GB
INFO  2020-12-02 21:40:32,905 [shard 0] iotune - /var/tmp/mnt passed sanity checks
This is a supported kernel version.
You current Scylla release is 3.2.2 the latest minor release is 4.2.1 go to http://www.scylladb.com for upgrade instructions
Traceback (most recent call last):
  File "/opt/scylladb/scripts/libexec/scylla_coredump_setup", line 41, in <module>
    if dist_name() == 'Ubuntu' and dist_ver() == '14.04':
  File "/opt/scylladb/scripts/scylla_util.py", line 429, in dist_name
    return platform.dist()[0]
AttributeError: module 'platform' has no attribute 'dist'
coredump setup failed.
```

Minimum way to reproduce:

use script below  "test.py"

```
import platform
print(platform.dist()[0])
print(platform.dist()[1])
```

run with embedded python 3.8   ( scylla-python3-3.8.5-0.20201108.4fb8ebccff.x86_64 )

`
/opt/scylladb/python3/bin/python3 test.py
`
this will fail with error:

```
Traceback (most recent call last):
 File “test.py”, line 2, in <module>
  print(platform.dist()[0])
AttributeError: module ‘platform’ has no attribute ‘dist’
```

this same test script succeed with standard Centos python and with python3.7 embedded with scylla-enterprise 2020.1.4

issue is with:

scylla 3.2.2-0.20200222.0b23e7145d0

```
ls -la /opt/scylladb/python3/bin/

lrwxrwxrwx. 1 root root  9 Dec 2 21:04 python3 -> python3.8
-rwxr-xr-x. 1 root root 273 Nov 8 12:48 python3.8

rpm -qf /opt/scylladb/python3/bin/python3.8
scylla-python3-3.8.5-0.20201108.4fb8ebccff.x86_64

```

While python version 3.7 from enterprise

scylla-enterprise-python3-3.7.7-0.20200826.7efb9d680e.x86_64.rpm

works OK.

**Comment by syuu1228** [comment]
Seems like you are using wrong version of scylla-python3, scylla-3.2 is shipped with scylla-python3-3.7.4.

Like this:
```
===============================================================================
 Package              Arch   Version                         Repository   Size
===============================================================================
Installing:
 scylla               x86_64 3.2.5-0.20200418.77500f9171.el7 scylla-3.2  6.5 k
Installing for dependencies:
 libreport-filesystem x86_64 2.1.11-53.el7.centos            base         41 k
 mdadm                x86_64 4.1-6.el7                       base        439 k
 scylla-conf          x86_64 3.2.5-0.20200418.77500f9171.el7 scylla-3.2   21 k
 scylla-jmx           noarch 3.2.5-20200418.857e433a3a       scylla-generic-3.2
                                                                         7.3 M
 scylla-kernel-conf   x86_64 3.2.5-0.20200418.77500f9171.el7 scylla-3.2  7.6 k
 scylla-python3       x86_64 3.7.4-0.20200418.77500f9171.el7 scylla-3.2   22 M
 scylla-server        x86_64 3.2.5-0.20200418.77500f9171.el7 scylla-3.2   38 M
 scylla-tools         noarch 3.2.5-20200418.09885dc0b3       scylla-generic-3.2
                                                                         157 k
 scylla-tools-core    noarch 3.2.5-20200418.09885dc0b3       scylla-generic-3.2
                                                                          27 M

Transaction Summary
===============================================================================
Install  1 Package (+9 Dependent packages)

Total download size: 95 M
Installed size: 299 M
Is this ok [y/d/N]:
```

You can try `yum remove scylla\*` and then install scylla-3.2 again.

**Comment by syuu1228** [comment]
Note: The error itself is already fixed at eabcb31503eb9896abeeaefe49cd548012dca001 and available from 4.2 and later.

**Comment by slivne** [comment]
scylla-3.3.2 is not supported.

The two Scylla Open Source versions currently supported are 4.1.X, 4.2.X 

Please upgrade
